src/components/AppHeader/AppHeader.tsx

```diff
@@ -7,6 +7,7 @@ import { Toolbar, useColorModeContext } from '@eten-lab/ui-kit';
 
 import { useAppContext } from '@/hooks/useAppContext';
 import { RouteConst } from '@/constants/route.constant';
+import { ColorThemes } from '../../constants/common.constant';
```

Comment:
`import { ColorThemes } from '../../constants/common.constant';`

to 

`import { ColorThemes } from '@/constants/common.constant';`